plugins/router/plugin.go

```diff
@@ -0,0 +1,54 @@
+package router
```

Comment:
Just out of curiosity, are you using this plugin? I don't see it listed in the config.

Reply:
Yes, i forgot to put it in the example config

Reply:
Again, just out of curiosity: Does your router have a different IP to the one of the DHCP server?

Reply:
No, it doesn't, but I think it can have a different IP to the one of the DHCP server.

Reply:
Do you need to set the `server_id` and the `router` in that case?

Reply:
My understanding of it is that the router field lists the available routers the client can connect to, while the server_id is used so the server knows it is supposed to respond to the DHCP request. So yes the server_id is always necessary.

Reply:
I thought the `server_id` plugin was also setting the `GatewayIPAddr`: https://github.com/insomniacslk/dhcp/blob/master/dhcpv4/dhcpv4.go#L69.

Disregards.